src/components/ResourceBrowser/ResourceList/NodeActionsMenu.tsx

```diff
@@ -181,38 +194,31 @@ export default function NodeActionsMenu({
             </PopupMenu>
             {showCordonNodeDialog && (
```

Comment:
Check if these modals needs to be moved in render block and short circuited since at one time only one modal should be open and if we move any one of them to query params it would cause issues

Reply:
moving them into a render function